src/index.html

```diff
@@ -9,10 +9,44 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The link to the stylesheet should have an `href` attribute that points to a `.css` file, not a `.scss` file. SCSS files are preprocessed and should be compiled into CSS. Make sure to compile your SCSS and link to the resulting CSS file.